### PR TITLE
Connect with API

### PR DESCRIPTION
**Pull request by Leandro-Barretoo**
In this PR I connected the redux store with an external API:
- Adding a book will display it on the page and add it to the API;
- Deleting a book will remove from the display and remove from the API as well;
- Book information will persist after reloading due fetching all book information from API;

In this stage, no styling was applied.
Thank you for the review.

### CODE REVIEW

File: src/components/AddForm.js [hunk 1]
```diff
@@ -5,16 +5,36 @@ import { addBook } from '../redux/books/books';
...
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      })
+        .then((response) => response);
     };
-    dispatch(addBook(newBook));
+
+    dispatch(postBooks());
     form.reset();
   };
```
Comment (DeliceLydia) [inline src/components/AddForm.js:39]: - [ ] When I try to post a book I can't see the name of the book, The Add book and remove book features should work the same as before even after the refactoring to use the API has been done.
ne

Please consider that

Comment (DeliceLydia) [reply src/components/AddForm.js:39]: - [ ] I suggest that you add author input as well

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Leandro-Barretoo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddForm.js (hunk 1 above)
Comment (Leandro-Barretoo) [reply src/components/AddForm.js:39]: Hey, the name of the book, category, and author is on top of the action buttons! Can I see the full display including the navbar. Tks.
The author input is not in the Zeplin design, so following the objective of this week of creating the design from that UI, I don't see it as a requirement, but something to add for practice.
Thank you

Comment (Leandro-Barretoo) [reply src/components/AddForm.js:39]: On my end, everything works fine, I can even see TSE book inputs.

Comment (DeliceLydia) [reply src/components/AddForm.js:39]: Hello,

bo

I am sorry its a bit hidden couldn't see book input

Comment (DeliceLydia) [reply src/components/AddForm.js:39]: Thanks for reaching out adding an author is a suggestion though!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @Leandro-Barretoo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**